controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -174,24 +201,97 @@ func (r *StorageConsumerReconciler) reconcilePhases() (reconcile.Result, error)
 	return reconcile.Result{}, nil
 }
 
+func (r *StorageConsumerReconciler) reconcileCephRadosNamespace(
+	consumerConfigMap *corev1.ConfigMap,
+	radosnamespace string,
```

Comment:
Your vars names and types need to match the usage you have in the context of this specific function (including order of importance) 
```suggestion
	radosNamespaceName string,
	additionalOwner client.Object,
```